tests/suites/test_suite_pk.function

```diff
@@ -1087,3 +1087,45 @@ exit:
     USE_PSA_DONE( );
 }
 /* END_CASE */
+
+/* BEGIN_CASE depends_on:MBEDTLS_PSA_CRYPTO_C:MBEDTLS_GENPRIME */
+void pk_psa_sign_ext( int pk_type, int parameter, int key_pk_type, int md_alg )
```

Comment:
```suggestion
/* See the description of pk_genkey() for the description of the `parameter` argument. */
void pk_psa_sign_ext( int pk_type, int parameter, int key_pk_type, int md_alg )
```
Just to help a bit when discovering the code.

Reply:
It can not be put here. report 
`generate_test_code.py: input error: file: suites/test_suite_pk.function - Test functions not found!`